src/arcam/fmj/__init__.py

```diff
@@ -878,6 +878,20 @@ class DecodeModeMCH(IntOrTypeEnum):
 }
 
 
+class IncomingVideoAspectRatio(IntOrTypeEnum):
+    UNDEFINED = 0x00
+    _4_3 = 0x01
+    _16_9 = 0x02
```

Comment:
Please add some more logical prefix,  not just underscore. So ASPECT_4_3 Would be better